docs/topics/readme/configmap-README.adoc

```diff
@@ -93,4 +93,4 @@ $ curl http://${app-name}-MY_PROJECT_NAME.LOCAL_OPENSHIFT_HOSTNAME/api/greeting?
 ----
 
 == More Information
-You can learn more about this booster and rest of the ${runtime} runtime in the link:${guideURL}[${runtime} Runtime Guide].
\ No newline at end of file
+You can learn more about this booster and rest of the ${runtime} runtime in the link:${guideURL}[${runtime} Runtime Guide].
```

Comment:
Either add "the" to "…this booster and **the** rest of the runtime…"
Or remove "rest of" from "…this booster and ~rest of~ the runtime…"